docs/reference/tmx-map-format.rst

```diff
@@ -61,6 +61,12 @@ might be useful for XML-namespacing anyway.*
 -  **nextobjectid:** Stores the next available ID for new objects. This
    number is stored to prevent reuse of the same ID after objects have
    been removed. (since 0.11)
+-  **objectalignment:** Controls the origins for tile and shape objects.
+   Valid values are ``unset``, ``top-left``, ``bottom-left``, and ``bottom-center``.
+   In isometric mode, ``bottom-left`` is treated the same as ``bottom-center``.
```

Comment:
I wasn't sure what to do about the isometric case.

There is no "bottom left" corner in isometric mode. If we use the left corner, then our name is wrong. If we use "true" bottom left (bottom left of bounding screen rect), then the origin is inside another tile, which makes no sense either.

Really, all that matters for object alignment in isometric mode is top vs bottom. If we had more options (like if we add top-right or bottom-right), I think the should also just become top or bottom corner.

I'm okay with whatever solution, I just wanted to avoid confusing names.